github_cog.py

```diff
@@ -0,0 +1,111 @@
+                msg += 'or contact a mod if the issue persists.'
+                await ctx.message.reply(msg)
+                return()
+
+            # Verify that the user has not already invited a GitHub account to the org.
+            with closing(studybot.db_conn.cursor()) as conn:
+                async with studybot.lock:
+                    query = 'SELECT github_id FROM github_invites WHERE discord_id = ?'
+                    conn.execute(query, (ctx.author.id, ))
+                    value = conn.fetchone()
+                    if value is None:
+                        # Verify that the user is not already in the org.
+                        if g_org.has_in_members(g_user):
+                            await ctx.message.reply('That user is already in the organization, and cannot be invited.')
+                            return()
+                        # Invite the user to the organization.
+                        else:
+                            g_org.invite_user(g_user)
+
+                        # Add the Discord ID and the GitHub ID to the database table
+                        query = 'INSERT INTO github_invites (discord_id, github_id) VALUES (?, ?)'
+                        conn.execute(query, (ctx.author.id, g_user.id))
+
+                        # Send a confirmation message in Discord.
+                        await ctx.message.reply(f'Invited {g_user.login} to the GitHub Organization.')
+                    else:
+                        msg = f'Cannot invite {g_user.login} to the organization, because you have already invited '
+                        msg += 'another GitHub account. Please contact a moderator if this is a mistake.'
+                        await ctx.message.reply(msg)
+                    studybot.db_conn.commit()
+
+    @commands.command(name='github-reset', help=studybot.help_dict.get('github-reset'))
+    @commands.has_role(studybot.admin_role_id)
+    async def reset_user(self, ctx, discord_user: discord.User = None):
+        if not discord_user:
+            await ctx.message.reply('You need to tag the user to reset their abilities.')
+            return()
+
+        # Find who the user previously invited to the org.
+        with closing(studybot.db_conn.cursor()) as conn:
+            async with studybot.lock:
+                query = 'SELECT github_id FROM github_invites WHERE discord_id = ?'
+                conn.execute(query, (discord_user.id, ))
+                value = conn.fetchone()
+
+                # If the user hasn't invited anyone
+                if value is None:
+                    await ctx.message.reply('This user has not invited anyone to the org.')
+                    return()
+
+                g = Github(studybot.TOKENS.get('GITHUB_API_KEY'))
+                # Get the GitHub user object.
+                try:
+                    g_user = g.get_user_by_id(int(value[0]))
+                except:
```

Comment:
What error does this raise when no user is there?

Reply:
It raises a general pygithub error, with the real information being in a returned json, which I don't know how to handle here.